src/signs/ui/display/SignLogogram/SignLogogram.tsx

```diff
@@ -0,0 +1,128 @@
+import { Logogram } from 'signs/domain/Sign'
+import LogogramWord from 'signs/ui/display/SignLogogram/LogogramWord'
+import MarkdownAndHtmlToHtml from 'common/MarkdownAndHtmlToHtml'
+import HelpTrigger from 'common/HelpTrigger'
+import _ from 'lodash'
+import ExternalLink from 'common/ExternalLink'
+import React from 'react'
+
+export default function SignLogograms({
+  signLogograms,
+  wordService,
+}: {
+  signLogograms: any
+  wordService: WordService
+}): JSX.Element {
+  return (
+    <Row>
+      <Col xs={'auto'}>Words (as logogram):</Col>
+      <Col>
+        <ul>
+          {signLogograms.map((logogram, index) => (
+            <li key={index}>
+              <SignLogogramEntry
+                logogram={logogram}
+                wordService={wordService}
+              />
+            </li>
+          ))}
+        </ul>
+      </Col>
+    </Row>
+  )
+}
+
+function SignLogogramEntry({
+  logogram,
+  wordService,
+}: {
+  logogram: Logogram
+  wordService: WordService
+}): JSX.Element {
+  return (
+    <Row>
+      <Col xs={5}>
+        {logogram.wordId.length > 0 && (
+          <SignLogogramEntryWords
+            logogramWordIds={logogram.wordId}
+            wordService={wordService}
+          />
+        )}
+        {logogram.logogram && (
+          <span>
+            (
+            <MarkdownAndHtmlToHtml
+              className={'signInformation__LogogramLogogram'}
+              container={'span'}
+              markdownAndHtml={logogram.logogram}
+            />
+            )
+          </span>
+        )}
+      </Col>
+      <Col>
+        {logogram.schrammLogogramme && (
+          <HelpTrigger
+            delay={{ show: 0, hide: 1500 }}
+            overlay={SignLogogramInfo(logogram.schrammLogogramme)}
+          />
+        )}
+      </Col>
+    </Row>
+  )
+}
+
+function SignLogogramEntryWords({
+  logogramWordIds,
+  wordService,
+}: {
+  logogramWordIds: readonly string[]
+  wordService: WordService
+}): JSX.Element {
+  return (
+    <>
+      {logogramWordIds.map((wordIdElem, index) => (
+        <span key={index}>
+          <LogogramWord wordId={wordIdElem} wordService={wordService} />
+          {index < logogramWordIds.length - 1 ? ', ' : ''}
+        </span>
+      ))}
+      &nbsp;
+    </>
+  )
+}
+
+function SignLogogramInfo(schrammLogogram: string): JSX.Element {
+  return (
+    <Popover
+      id={_.uniqueId('LogogramInfo-')}
+      title="Logogram Info"
+      className={'signDisplay__LogogramInfo'}
+    >
+      <Popover.Content>
+        <MarkdownAndHtmlToHtml
+          className="text-center my-1"
+          markdownAndHtml={schrammLogogram}
+        />
+        <div className="text-center mt-3">
+          <small>
+            From W. Schramm,{' '}
+            <em>Akkadische Logogramme. Zweite, revidierte Auflage.&nbsp;</em>
+            Göttinger Beiträge zum Alten Orient 5. Göttingen: Universitätsverlag
+            Göttingen,&nbsp;
+            <sup>2</sup>2010. (CC BY-ND 3.0).
```

Comment:
Add a link to the license.

Can we use the data or is our use a derivate?

Reply:
i will ask enrique.

Reply:
@ejimsan you can probably answer this.

Reply:
It should be allowed — I double checked with the author and he said it is fine.